compiler/src/iree/compiler/Codegen/LLVMGPU/test/attention_transform_spec.mlir

```diff
@@ -30,10 +30,10 @@ module attributes { transform.with_named_sequence } {
     // Tile and decompose attention
     // ==========================================
     %attention4 = transform.structured.match ops{["iree_linalg_ext.attention"]} in %variant_op : (!transform.any_op) -> !transform.any_op
-    %acc_fill, %max_fill, %sum_fill, %inner_loop, %final_scaling, %last_truncate, %blocked_attention = transform.tile_attention %attention4 :
+    %acc_fill, %max_fill, %sum_fill, %inner_loop, %final_scaling, %last_truncate, %blocked_attention = transform.tile_attention %attention4 {tile_size = 0} :
```

Comment:
Is there a way to make the tile_size optional instead of 0?

Reply:
Let me look into it.